app/components/PlayQueue/QueueItem/styles.scss

```diff
@@ -1,8 +1,17 @@
 @import '../../../vars';
 
+@keyframes appear {
```

Comment:
Thanks for this change, I implemented this originally when I had no idea how to use css animations.